armi/utils/tests/test_codeTiming.py

```diff
@@ -28,52 +28,101 @@ def tearDown(self):
         codeTiming._Timer._frozen = False
         codeTiming.MasterTimer._instance = None
 
-    def test_method_definitions(self):
+    def test_methodDefinitions(self):
+        """Test that the timer decorators do not interupt the code being decorated."""
```

Comment:
would be great to have a test that shows that the decorator actually _do_ something too. thoughts?

Reply:
That's the rest of the unit tests in the file, right?

Reply:
The rest of the tests cover `MasterTimer` functionality, yes. I suppose I meant more that we should probably show that the if a decorator is used it makes it into the report (or is otherwise useful). I don't see that tested anywhere. I think the args options for the decorator aren't fully tested either - at least not in this test module.

Reply:
If it helps, this PR gives the `codeTiming.py` file 98.51% code coverage: https://coveralls.io/jobs/163599512/source_files/2289616344

I can look into adding more testing.

Reply:
Having that high coverage metric is great! If you're confident that the decorator works as intended (adding it to a function does indeed populate in the table with an expected title) with the existing tests, great. I just didn't see one explicitly but I am happy to convinced otherwise

Reply:
Okay, I added a test to show that using the decorator works (the report is populated).